src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -451,8 +445,8 @@ export default defineComponent({
       this.channelName = this.data.author ?? null
       this.channelId = this.data.authorId ?? null
 
-      if (this.data.isRSS && this.historyIndex !== -1) {
-        this.duration = formatDurationAsTimestamp(this.historyCache[this.historyIndex].lengthSeconds)
+      if (this.data.isRSS && typeof this.historyEntryExists) {
```

Comment:
Is there a reason typeof is being used here?

Reply:
forgot to remove it in my last commit 🙈